example/lib/styles/colors.dart

```diff
@@ -146,6 +141,6 @@ class AppColors {
   static const Color warningHover = Color(0xffFFAD14);
   static const Color warningHoverDark = Color(0xffFFA90A);
 
-  static const Color disabledBackgroundHover = Color(0x331A1B1F);
+  static const Color disabledBackgroundHover = Color.fromARGB(51, 26, 27, 31);
```

Comment:
Same as earlier 51 can be converted to hex and the format used in other places can be used for consistency.

Reply:
Done.